presto-main/src/test/java/com/facebook/presto/operator/TestHashAggregationOperator.java

```diff
@@ -244,6 +244,51 @@ public void testHashAggregationWithGlobals(boolean hashEnabled, long memoryLimit
         assertOperatorEqualsIgnoreOrder(operatorFactory, driverContext, input, expected, hashEnabled, Optional.of(groupByChannels.size()));
     }
 
+    @Test(dataProvider = "hashEnabledAndMemoryLimitBeforeSpillValues")
```

Comment:
where do you define the `hashEnabledAndMemoryLimitBeforeSpillValues` data provider?